rules/aws_cloudtrail_rules/aws_cloudtrail_unsuccessful_mfa_attempt.py

```diff
@@ -0,0 +1,16 @@
+from panther_base_helpers import deep_get, aws_rule_context
+
+def rule(event):
```

Comment:
I believe this rule should provide `eventSource` and `eventName` filters before going into the evaluation below.

Reply:
This has been addressed.